tests/js/JSONFormatBearTest.py

```diff
@@ -41,6 +42,12 @@
 # This will generate a line with 79 characters
 max_line_length_file2 = '{\n    "string": "' + 'a' * 63 + '"\n}'
 
+# This file is for checking the appropriate message displayed
+# to the user based on the json_sort value
+test_file5 = '{\n', '    "a":   5\n', '}\n'
+
+correct_test_file5 = '{\n', '    "a": 5\n', '}\n'
```

Comment:
Some relevant names to the test files would be better.

Reply:
Will do in my next commit..